Documentation/user-manual.txt

```diff
@@ -2759,7 +2771,14 @@ A dive log or part of it can be saved in several formats:
 * _CSV dive profile_, that includes a large amount of detail for each dive, including
   the depth profile, temperature and pressure information of each dive.
 
-* _dive profile_, saves the Subsurface dive profile as a .png file
+* _Dive profile_, which enables easy sharing of the dive profile using social media.
+  This option writes the image of the dive profile(s) in a _.png_ graphics
+  format to the folder from which _Subsurface_ was started. The saved image
```

Comment:
It should save "to the file system", specifically to the folder selected in the file chooser (see below)

Reply:
Done.